### DISCUSSION TITLE
How to define a shell discrete rigid part?

### DISCUSSION THREAD

**Discussion by Majid-MHZ** [OpenRadioss/OpenRadioss#2702] [Q&A]
Hello everyone,

I am converting my Abaqus input file to a compatible one with OpenRadioss. I have modeled a punch and box as a shell discrete rigid part and allocated a reference point to each of them separately. I want to know if I can create a shell discrete rigid part in OpenRidioss. If so, please help me how I can manage it. 

Best,
Majid

**Comment by MarianBulla** [comment]
Hi @Majid-MHZ ,
Please review the /RBODY card.
You can create your rigid part using shells. Then add this part into the GroupID for the nodes, so they become a rigid body.
The main node will be the controlling node of that body, which can also track the forces comming from the secondary nodes in the timehistory file.

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/1af8d5ee-2a21-4bb1-bcc4-53e66d5f8722)

Hope it helps you ?

Thanks and best regards,
Marian
